modules/test/playwright/tests/client-extension-web/editorConfigContributor.spec.ts

```diff
@@ -6,39 +6,34 @@
 import {expect, mergeTests} from '@playwright/test';
 
 import {apiHelpersTest} from '../../fixtures/apiHelpersTest';
+import {featureFlagsTest} from '../../fixtures/featureFlagsTest';
+import {isolatedSiteTest} from '../../fixtures/isolatedSiteTest';
 import {loginTest} from '../../fixtures/loginTest';
+import {liferayConfig} from '../../liferay.config';
+import getRandomString from '../../utils/getRandomString';
+import getPageDefinition from '../layout-content-page-editor-web/utils/getPageDefinition';
+import getWidgetDefinition from '../layout-content-page-editor-web/utils/getWidgetDefinition';
 import {clientExtensionsPageTest} from './fixtures/clientExtensionsPageTest';
+import {editorSamplePageTest} from './fixtures/editorSamplePageTest';
 import {newEditorConfigContributorPageTest} from './fixtures/newEditorConfigContributorPageTest';
 
 export const test = mergeTests(
 	apiHelpersTest,
 	clientExtensionsPageTest,
+	editorSamplePageTest,
+	featureFlagsTest({
+		'LPS-178052': true,
+		'LPS-186870': true,
+	}),
+	isolatedSiteTest,
 	loginTest(),
 	newEditorConfigContributorPageTest
 );
 
-test('Editor config contributor client extension is disabled if FF is set to false', async ({
```

Comment:
I removed test that checks behavior is FF is not set. We added this very early on, but thinking about it now, I believe the pattern is unnecessary and disruptive. FFs are transient, they will disappear in time. When that happens, it will not only negate effort to write these tests, but also break them. It's better to focus on behavior when FFs are enabled. Such tests will not break when FF is removed, they will only contain unnecessary FF fixture.

Reply:
makes sense. There's no pattern to dictate when this shall be removed. Imo, this should be part of the FF removal commit to make things consistent. I agree test should not stay forever.